### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by tusindfryd**

### CODE REVIEW

File: doc/Language/create-cli.pod6 [hunk 1]
```diff
@@ -12,7 +12,7 @@ The default command line interface of Raku scripts consists of three parts:
 =head2 Parsing the command line parameters into a L<capture|/type/Capture>
 
 This looks at the values in L<@*ARGS|/language/variables#index-entry-@*ARGS>,
-interprets these according to some policy, and creates a L<Capture/type/Capture>
+interprets these according to some policy, and creates a L<capture|/type/Capture>
```
Comment (JJ) [inline doc/Language/create-cli.pod6]: Can you please keep the capital for Capture here?

File: doc/Language/operators.pod6 [hunk 2]
```diff
@@ -1889,7 +1889,7 @@ is smaller than or equal to the second.
 
 Numeric less than or equal to operator.
 
-Equivalent to L«<=|routine/<=», at codepoint U+2264 (LESS-THAN OR EQUAL TO).
+Equivalent to L«<=/routine/<=», at codepoint U+2264 (LESS-THAN OR EQUAL TO).
```
Comment (JJ) [inline doc/Language/operators.pod6]: I don't think this this should be changed actually

File: doc/Language/create-cli.pod6 (hunk 1 above)
Comment (tusindfryd) [reply doc/Language/create-cli.pod6]: Done :)

File: doc/Language/operators.pod6 (hunk 2 above)
Comment (tusindfryd) [reply doc/Language/operators.pod6]: Why? The first one is not working for me.

Comment (JJ) [reply doc/Language/operators.pod6]: But you're changing a text-url separator (`|`) to a slash for directory separation. That one is not going to work no matter what. Maybe you should leave `|/` instead?

Comment (tusindfryd) [reply doc/Language/operators.pod6]: Oh, that's right. My bad. Changed now

**Comment by JJ** [comment]
Thanks for doing this, you have now permission to write in the repository. Congratulations!

**Pull request merged**